pkg/reconciler/v1alpha1/taskrun/taskrun.go

```diff
@@ -166,8 +162,20 @@ func (c *Reconciler) reconcile(ctx context.Context, tr *v1alpha1.TaskRun) error
 	// get build the same as the taskrun, this is the value we use for 1:1 mapping and retrieval
 	build, err := c.BuildClientSet.BuildV1alpha1().Builds(tr.Namespace).Get(tr.Name, metav1.GetOptions{})
 	if errors.IsNotFound(err) {
+		pvc, err := c.KubeClientSet.CoreV1().PersistentVolumeClaims(tr.Namespace).Get(tr.Name, metav1.GetOptions{})
+		if errors.IsNotFound(err) {
+			// Create a persistent volume claim to hold Build logs
+			pvc, err = c.createPVC(tr)
+			if err != nil {
+				return fmt.Errorf("Failed to create persistent volume claim %s for task %q: %v", tr.Name, err, tr.Name)
+			}
+		} else if err != nil {
+			c.Logger.Errorf("Failed to reconcile taskrun: %q, failed to get pvc %q; %v", tr.Name, tr.Name, err)
+			return err
+		}
+
```

Comment:
you can probably rebase at this point  since #167 is merged and this review will be a bit clearer